### PR TITLE
scripts/InstallScripts: set root password for non-desktop installs

### PR DESCRIPTION
**Pull request by austin987**
This refactors things a bit to make it (IMO) a bit clearer.

Tested with a non-desktop install and with Xfce.

### CODE REVIEW

**Comment by austin987** [comment]
Should I rebase this?

**Comment by SolidHal** [comment]
@austin987 sorry for the late reply. I'm about to merge in arm64 support, so if you're still interested in rebasing this I would wait until after that. If you don't feel like it, I'll get around to rebasing it :)

**Comment by austin987** [comment]
I'll leave it to you, I'm working on some other things this weekend ;)

File: scripts/InstallScripts/InstallPrawnOS.sh [hunk 1]
```diff
@@ -162,11 +162,12 @@ install() {
     while true; do
         read -p "Install a desktop environment and the supporting packages? [Y/n]" ins
         case $ins in
-            [Yy]* ) install_packages $INSTALL_MOUNT; break;;
```
[the remark this answers is not in the document]
Comment (SolidHal) [reply scripts/InstallScripts/InstallPrawnOS.sh:165]: I realized I didn't ever submit this review, so here it is :/

**Comment by austin987** [comment]
@SolidHal updated.

**Comment by SolidHal** [comment]
Thanks for rebasing this @austin987!

**Pull request merged**